src/Agent/NewRelic/Agent/Core/Metrics/MetricNames.cs

```diff
@@ -1241,18 +1270,18 @@ public static string GetSupportabilityLogForwardingEnabledWithFrameworkName(stri
 
         #endregion
 
-        #region Open Telemetry Bridge
+        #region Open Telemetry
 
-        private const string OpenTelemetryBridge = "OpenTelemetryBridge";
+        private const string OpenTelemetry = "OpenTelemetry";
```

Comment:
This value needs to be updated to `OpenTelemetryBridge` and references to that string above should use this variable instead.